protocol/communities/persistence_test.go

```diff
@@ -375,7 +375,7 @@ func (s *PersistenceSuite) TestGetCommunityTokens() {
 	s.Require().NoError(err)
 	s.Require().Len(tokens, 0)
 
-	token := CommunityToken{
+	token721 := CommunityToken{
```

Comment:
No biggie, but I think if we **do** make the effort to rename this variable here to make it semantically more meaningful, we should probably call it at least something like `tokenERC721` or so

Reply:
Done